src/output/fmt.rs

```diff
@@ -8,6 +8,7 @@ use std::{
 #[derive(Clone, Debug)]
 pub(crate) struct Colorizer {
     use_stderr: bool,
+    #[allow(unused)]
```

Comment:
i don't love doing this, but this seems to be the least painful approach for now.